content/en/docs/concepts/configuration/secret.md

```diff
@@ -24,6 +24,16 @@ a password, a token, or a key. Such information might otherwise be put in a
 Pod specification or in an image. Users can create Secrets and the system
 also creates some Secrets.
 
+{{< caution >}}
+Kubernetes Secrets are, by default, stored as unencrypted base64-encoded
+strings. By default, once they are written to Kubernetes, they can be retrieved
```

Comment:
nit: line 29, Does it make sense to remove "once they are written to Kubernetes"?
[Once written], Secrets can be retrieved as plain text by anyone with API access or anyone with access ...